wp-content/mu-plugins/pantheon/pantheon-updates.php

```diff
@@ -42,11 +42,16 @@ function _pantheon_curl_cached( $api_url ) {
 
 // Replace WordPress core update nag EVERYWHERE with our own notice (use git upstream)
 function _pantheon_upstream_update_notice() {
+	$update_type = 'new WordPress version';
+	$data = _pantheon_curl_cached( 'https://api.live.getpantheon.com/sites/self/code-upstream-updates' );
+	if ( ! empty( $data['remote_url'] ) && false === stripos( $data['remote_url'], '/pantheon-systems/' ) ) {
+		$update_type = 'Pantheon Custom Upstream update';
+	}
 
     ?>
     <div class="update-nag">
 		<p style="font-size: 14px; font-weight: bold; margin: 0 0 0.5em 0;">
-			A new WordPress version is available! Please update from <a href="https://dashboard.pantheon.io/sites/<?php echo $_ENV['PANTHEON_SITE']; ?>">your Pantheon dashboard</a>.
+			A <?php echo $update_type; ?> is available! Please update from <a href="https://dashboard.pantheon.io/sites/<?php echo $_ENV['PANTHEON_SITE']; ?>">your Pantheon dashboard</a>.
 		</p>
 		For details on applying updates, see the <a href="https://pantheon.io/docs/upstream-updates/" target="_blank">Applying Upstream Updates</a> documentation. <br />
 		If you need help, open a support chat on Pantheon.
```

Comment:
This line should be changed to `If you need help, contact an administrator for your Pantheon organization.` for custom upstreams. Maybe use a variable `$update_help_text` similar to how you are using `$update_type`?

Reply:
Ah, my bad — I missed this. Updated in f222ad6